app/src/main/sentences/fr/lyrics.dslf

```diff
@@ -0,0 +1,5 @@
+lyrics: high
+trouve|trouvez|cherche|cherchez|recherche|recherchez|charge|chargez|consulte|consultez (pour? moi)? les?|le? paroles|texte (de (la musique|chanson)?)?  .song.;
```

Comment:
If a single word in an or-list is optional, obviously the whole or-list becomes optional if you think about it, so I allowed putting `?` only at the end of the or-list to increase consistency  
```suggestion
trouve|trouvez|cherche|cherchez|recherche|recherchez|charge|chargez|consulte|consultez (pour? moi)? les|le? paroles|texte (de (la musique|chanson)?)?  .song.;
```